aidants_connect_web/tests/test_models.py

```diff
@@ -91,6 +92,7 @@ def test_saving_and_retrieving_mandat(self):
             birthplace=26934,
             birthcountry=99100,
             email="ned@flanders.com",
+            sub="1234",
```

Comment:
You could test that if there are two instances with the same sub, it throws an error